linux/draftInstall.sh

```diff
@@ -68,7 +68,7 @@ checkLatestVersion() {
 # for that binary.
 downloadFile() {
   DIST="$PROJECT_NAME-$TAG-$OS-$ARCH.tar.gz"
-  DOWNLOAD_URL="https://azuredraft.blob.core.windows.net/draft/$DIST"
+  DOWNLOAD_URL="https://azuredraft.blob.core.windows.net/draft/draft-v0.16.0-linux-amd64.tar.gz" #short-term fix 
```

Comment:
If I understand correctly, the PROJECT_NAME has changed from draft-latest to draft-v0.16.0 and we might want to generate this as a long-term fix?

Reply:
I think the PROJECT_NAME is ‘draft’. We set the $TAG to v0.16.0 which is the latest tag that worked last time. Generally, we grab our tag from this line (https://github.com/Azure/CloudShell/blob/master/linux/draftInstall.sh#L57) but for some reason it's no longer work. Robin said the github repo isn't publicly available and we are referring to a public archive that's fairly out of date. The active repo looks to be internal to Microsoft accounts only so curl/wget isn't going to be able to download the files. So we need to ask for some help from the team who owns the draft repo to find a different way to do it for the long term fix.